src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/InitialConsiderationService.java

```diff
@@ -62,6 +62,7 @@ public class InitialConsiderationService {
     private static final String RESPONDENT_MISSING = String.format(RESPONDENT_NAME, "", "", "");
     private static final String DOCGEN_ERROR = "Failed to generate document for case id: %s";
     private static final String IC_OUTPUT_NAME = "Initial Consideration.pdf";
+    private static final String HEARING_STATUS_LISTED = "Listed";
```

Comment:
May be you can use this from ecm-common -> Constants.java, this status const is already declared there.

Reply:
Agreed. Fixed